File: Technology Fundamentals/02.Exercises - Intro and Basic Syntax/JavaScript/01.Exercise/triangleOfNumbers.js
function solve(number){
    let output = "";
    for(let row = 1; row <= number; row++){
        for(let col = 1; col <= row; col++){
            output += row + " ";
        }
        output += "\n";
    }
    console.log(output);
}
solve(5);

File: C# Fundamentals May 2019/02.Exercises - Intro and Basic Syntax/Java/src/TriangleOfNumbers.java
import java.util.Scanner;

public class TriangleOfNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int number = Integer.parseInt(scanner.nextLine());

        for(int row = 1; row <=number; row++){
            for(int col = 1; col <= row; col++){
                System.out.print(row + " ");
            }
            System.out.println();
        }
    }
}
